frontend/src/api/preferences/getAllUserPreference.ts

```diff
@@ -0,0 +1,18 @@
+import axios from 'api';
+import { ErrorResponse, SuccessResponse } from 'types/api';
+import { GetAllUserPreferencesResponseProps } from 'types/api/preferences/userOrgPreferences';
+
+const getAllUserPreferences = async (): Promise<
+	SuccessResponse<GetAllUserPreferencesResponseProps> | ErrorResponse
```

Comment:
the success response needs to be updated for all APIs, are we targeting that in follow up PR for consumption ?

Reply:
The consumption will be done as part of Onboarding V2. Will update the success response to be generic JS Object.